File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        if(s.length() % 2 == 0) {
            //절반 나눈 값 -1 ~ 절반 나눈 값 +1
            answer = s.substring(s.length()/2-1, s.length()/2+1);
        } else {
            //절반 인덱스 ~ 절반 + 1 인덱스 추출
            answer = s.substring(s.length()/2, s.length()/2+1);
        }
        
        return answer;
    }
}

File: 프로그래머스/Lv.1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.kt
class Solution {
    fun solution(s: String): String {
        return if (s.length % 2 == 0) {
            //절반 나눈 값 -1 ~ 절반 나눈 값 +1
            s.substring(s.length / 2 - 1, s.length / 2 + 1)
        } else {
            //절반 인덱스 ~ 절반 + 1 인덱스 추출
            s.substring(s.length / 2, s.length / 2 + 1)
        }
    }
}